packages/dd-trace/src/encode/0.4.js

```diff
@@ -29,7 +29,7 @@ class AgentEncoder {
   }
 
   makePayload () {
-    const prefix = Buffer.alloc(5)
+    const prefix = Buffer.allocUnsafe(5)
```

Comment:
This is changed because we're always using a 5-byte prefix, right?

Reply:
It's changed because `allocUnsafe` is faster and is able to use Node's buffer pool.

Reply:
Right, but presumably there was some reason to use `alloc` before. Or maybe there wasn't and it was a mistake?

Reply:
There wasn't and it was a mistake. I didn't consider pool usage when originally writing this line.